### PR TITLE
Add buttons to trigger unsymbolicated crashes

### PR DESCRIPTION
**Pull request by Jamminroot**
Things to consider before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [X] Are tests passing locally?
* [X] Are the files formatted correctly?
* [ ] Did you add unit tests if this modifies the Windows code?
* [X] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

This PR adds buttons to trigger unsymbolicated crashes to Demo and Puppet apps of UWP and WPF.
Adresses first part of @guperrot's [comment](https://github.com/microsoft/appcenter-sdk-dotnet/pull/1278#pullrequestreview-339631754)
## Related PRs or issues
#1278

### CODE REVIEW

File: AppCenter-Windows.sln [hunk 1]
```diff
@@ -970,6 +981,34 @@ Global
 		{2FFFF8C2-E190-4B83-9AD8-19F0E2AC6088}.Release|x64.Build.0 = Release|Any CPU
 		{2FFFF8C2-E190-4B83-9AD8-19F0E2AC6088}.Release|x86.ActiveCfg = Release|Any CPU
 		{2FFFF8C2-E190-4B83-9AD8-19F0E2AC6088}.Release|x86.Build.0 = Release|Any CPU
+		{A42BE92C-616D-4333-9F6A-D05BA547BFC6}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{A42BE92C-616D-4333-9F6A-D05BA547BFC6}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{A42BE92C-616D-4333-9F6A-D05BA547BFC6}.Debug|ARM.ActiveCfg = Debug|Any CPU
+		{A42BE92C-616D-4333-9F6A-D05BA547BFC6}.Debug|ARM.Build.0 = Debug|Any CPU
+		{A42BE92C-616D-4333-9F6A-D05BA547BFC6}.Debug|ARM64.ActiveCfg = Debug|Any CPU
+		{A42BE92C-616D-4333-9F6A-D05BA547BFC6}.Debug|ARM64.Build.0 = Debug|Any CPU
+		{A42BE92C-616D-4333-9F6A-D05BA547BFC6}.Debug|iPhone.ActiveCfg = Debug|Any CPU
```
Comment (Jamminroot) [inline AppCenter-Windows.sln]: Not sure if we actually need those iPhone targets here

Comment (MatkovIvan) [reply AppCenter-Windows.sln]: No need for it, please remove

File: Apps/Contoso.WPF.Puppet/MainWindow.xaml.cs [hunk 2]
```diff
@@ -244,6 +245,11 @@ private async void CrashInsideAsyncTask_Click(object sender, RoutedEventArgs e)
             }
         }
 
+        private void ClassLibraryException_Click(object sender, RoutedEventArgs e)
+        {
+            CrashUtils.BackgroundExceptionTask().RunSynchronously();
```
Comment (guperrot) [inline Apps/Contoso.WPF.Puppet/MainWindow.xaml.cs]: There is no symbol on WPF, do we really the button on WPF apps?

Comment (guperrot) [reply Apps/Contoso.WPF.Puppet/MainWindow.xaml.cs]: I guess it does not hurt though.

File: Apps/Contoso.UWP.Demo/MainPage.xaml.cs [hunk 3]
```diff
@@ -42,6 +43,10 @@ private async void ThrowException(object sender, RoutedEventArgs e)
             // Handled errors in the forms app never hit that case because we need to use v16299 there.
             await GenerateComplexException(2);
         }
+        private void ClassLibraryException_Click(object sender, RoutedEventArgs e)
+        {
+            CrashUtils.BackgroundExceptionTask().RunSynchronously();
```
Comment (guperrot) [inline Apps/Contoso.UWP.Demo/MainPage.xaml.cs]: It's not handled and tracked as an error if the toggle is enabled

File: Apps/Contoso.UWP.Demo/MainPage.xaml.cs [hunk 4]
```diff
@@ -42,6 +43,10 @@ private async void ThrowException(object sender, RoutedEventArgs e)
             // Handled errors in the forms app never hit that case because we need to use v16299 there.
             await GenerateComplexException(2);
         }
+        private void ClassLibraryException_Click(object sender, RoutedEventArgs e)
```
Comment (guperrot) [inline Apps/Contoso.UWP.Demo/MainPage.xaml.cs]: remove _Click as we don't have it on other buttons in this app

File: AppCenter-Windows.sln [hunk 5]
```diff
@@ -129,6 +138,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.AppCenter.Test.Wi
 EndProject
 Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "Microsoft.AppCenter.Test.WindowsDesktop.Shared", "Tests\Microsoft.AppCenter.Test.WindowsDesktop.Shared\Microsoft.AppCenter.Test.WindowsDesktop.Shared.shproj", "{09F38506-8DEF-431D-A749-DF7E73A719F0}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Contoso.UtilClassLibrary", "Apps\Contoso.UtilClassLibrary\Contoso.UtilClassLibrary.csproj", "{A42BE92C-616D-4333-9F6A-D05BA547BFC6}"
```
Comment (guperrot) [inline AppCenter-Windows.sln]: Can actually reference it in Forms version of the apps and use it. That would be good as the demo used by CTI is the forms one.

File: AppCenter-Windows.sln (hunk 1 above)
Comment (guperrot) [reply AppCenter-Windows.sln]: Its VS doing that automatically, happened again on Mac solution when I reused the code for Xamarin forms. I guess it's ok...

**Pull request merged**